README.md

```diff
@@ -1,37 +1,97 @@
+| ------------ | ------------------------------------------------------------ | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| VLCKit       | ![Platform](https://img.shields.io/cocoapods/p/VLCKit.svg?style=flat) | ![CircleCI](https://img.shields.io/circleci/project/github/videolan/vlckit.svg) | [![VLCKit is CocoaPods Compatible](https://img.shields.io/cocoapods/v/VLCKit.svg)](https://cocoapods.org/pods/VLCKit) |
+| MobileVLCKit | ![Platform](https://img.shields.io/cocoapods/p/MobileVLCKit.svg?style=flat) | ![CircleCI](https://img.shields.io/circleci/project/github/videolan/vlckit.svg) | [![MobileVLCKit is CocoaPods Compatible](https://img.shields.io/cocoapods/v/MobileVLCKit.svg)](https://cocoapods.org/pods/MobileVLCKit) |
+| TVVLCKit     | ![Platform](https://img.shields.io/cocoapods/p/TVVLCKit.svg?style=flat) | ![CircleCI](https://img.shields.io/circleci/project/github/videolan/vlckit.svg) | [![TVVLCKit is CocoaPods Compatible](https://img.shields.io/cocoapods/v/TVVLCKit.svg)](https://cocoapods.org/pods/TVVLCKit) |
+
+- [Features](#features)
+- [Use-case](#use-case)
+- [Requirements](#requirements)
+- [Installation](#installation)
+- [Build](#build)
+  - [Default](#default)
+  - [Build with your own VLC repository](#build-with-your-own-vlc-repository)
+- [Contribute](#contribute)
+- [FAQ](#faq)
+- [Communication](#communication)
+- [License](#license)
+- [Further reading](#further-reading)
 
 **VLCKit** is a generic multimedia library for any audio or video playback needs on macOS, iOS and tvOS.
 
-It is based on **libVLC**, the engine of the popular media player *VLC*.
+## Features
 
-It supports playback, but also  active streaming and media to file conversations on the Mac.
+- Based on **libVLC**, the engine of the popular media player *VLC*.
+- Supports playback, active streaming, and media to file conversations on the Mac.
+- Open-source software licensed under [LGPLv2.1] or later, available in source code and binary form from [VideoLAN's website].
+- Easily integratable via [CocoaPods].
 
-It is open-source software licensed under LGPLv2.1 or later, available in source code and binary form from the [VideoLAN website].
+## Use-case
 
-You can also integrate VLCKit and its mobile version MobileVLCKit easily via [CocoaPods].
+When will you need VLCKit? Frankly, you will need it whenever you need to play media not supported by QuickTime / AVFoundation or if you require more flexibility. 
 
+Here are some other common use-cases
 
-## Use-case
+- Playing something else besides H264/AAC files or HLS streams
+- Need subtitles beyond QuickTime’s basic support for Closed Captions
+- Your media source is neither your mobile device nor a basic HTTP server, but a live stream hailing from some weird media server or even a raw DVB signal broadcasted on a local network 
+- and more!
+
+## Requirements
+
+- iOS 8.0 + / macOS 10.9+ / tvOS 9.0+
```

Comment:
tvOS 10.2 from what we have in the News unless felix commits weren't cherry-picked